packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -178,7 +178,6 @@ ts_library(
         "@npm//sass",
         "@npm//sass-loader",
         "@npm//semver",
-        "@npm//source-map",
```

Comment:
Should also be removed from https://github.com/angular/angular-cli/blob/8be19dada4dc0593a95f7309790f9baf8d213ef9/packages/angular_devkit/build_angular/BUILD.bazel#L181

Reply:
I think that's the line changed here unless I missed another location?